modules/service/src/main/resources/app.conf

```diff
@@ -73,16 +73,11 @@ app {
 
   # OIDC connectivity settings
   oidc {
-    # The OIDC server discovery URI used for discovering its endpoints
-    discovery-uri = ${?OIDC_DISCOVERY_URI}
-    # The platform client id used for authenticating to the server
-    client-id = ${?OIDC_CLIENT_ID}
-    # The platform client secret used for authenticating to the server
-    client-secret = ${?OIDC_CLIENT_SECRET}
-    # The authorization scope
-    scope = ["openid"]
-    # This application's token uri
-    token-uri = ${app.http.public-uri}"/"${app.http.prefix}"/oauth2/token"
-    token-uri = ${?TOKEN_URI}
+    authorize-endpoint = "http://localhost:8080/oauth2/authorize"
```

Comment:
please add some comments on the values that need to be set on these configuration elements